### DISCUSSION TITLE
Is base64 supported?

### DISCUSSION THREAD

**Discussion by InverMN** [ory/kratos#1517] [Q&A] [answered]
Hello! When I am trying to provide identity schema as base64 and navigate to `http://127.0.0.1:4434/schemas/default` address I get that response (some code ommited for readability):
```json
{"error":{"code":500,"status":"Internal Server Error","reason":"The file for this JSON Schema ID could not be found or opened. This is a configuration issue.","debug":"Get \"base64://e(...)Q==\": unsupported protocol scheme \"base64\"","message":"An internal server error occurred, please contact the system administrator"}}
```
Any ideas why I get the error?

**Comment by Benehiko** [comment, marked as answer]
Hi @InverMN 

I have looked into this and could reproduce the error.
It seems the problem is this specific endpoint not reading the base64 url and only returning results which have a valid url (http or file) as shown [here](https://github.com/ory/kratos/blob/647eb1e66850c67e539d0338cca6cb8ae476ee55/schema/handler.go#L83-L98). The registration/login etc uses the base64 fine it seems.

I have created an issue to track this https://github.com/ory/kratos/issues/1529
